### DISCUSSION TITLE
Allow for nested lists/folders

### DISCUSSION THREAD

**Discussion by flamingjupiter** [Kovah/LinkAce#654] [Ideas & Feature Requests]
Sometimes, and for some things, it's easier to have nested lists/folders of links, than using tags; one-level-only lists; or a combination of the two. 

Example: imagine every "how to X" link or list of links you save. It's much easier to put all of it in a single list/folder of "how to guides" than to tag everyone with at least two tags (a "how to" and a "how to specifically-this" tag) or every list. 

Also, a way to browse through your saved links, just like a file system structure (hierarchy of folders/lists, and links organized in a specific order) instead of only being able to search things would be nice.

**Comment by flamingjupiter** [comment]
On the last paragraph, chrome's bookmark manager is also an example of what I'm talking about.

**Comment by vfrex** [comment]
I'm still thinking through how to approach this with tags/lists, but for reference, I have ~1,800 bookmarks, a significant portion of which are organized into Folder->Subfolder, and sometimes Folder->Subfolder->Subfolder->subfolder. For example, Computers ->Programming -> [Language] -> [Libraries, Learning Resources]

I have links and categories where tags and lists work better than folders/subfolders, but some stuff seems to work well with nesting folders.

**Comment by yamikeigo** [comment]
this would be a great addition IMO
